ci/ci_setup.sh

```diff
@@ -14,3 +14,7 @@ rm -rf spec/reports # no stale spec reports from previous executions
 
 # Setup the environment
 rake bootstrap # Bootstrap your logstash instance
+
+# Set up some general options for the rspec runner
+echo "--order rand" > .rspec
+echo "--format CI::Reporter::RSpec" >> .rspec
```

Comment:
The reporters is something from our jenkins, test task should not be dependant on formatters that our Jenkins like to use ;-) ... but as I said this is my point, jejejej.